.github/workflows/autobuild.yml

```diff
@@ -314,7 +314,8 @@ jobs:
       - name:                       Checkout code
         uses:                       actions/checkout@v4
         with:
-          submodules:               true
+          # only Android needs the oboe submodule, so don't fetch it for other builds
```

Comment:
Being picky, this should be part of a separate change, really.

Reply:
Yes, I did wonder whether to bite the bullet and raise a whole new PR for the one line. Can do if you prefer.

Reply:
OK, I've moved it to #3289